helm_deploy/values-dev.yaml

```diff
@@ -16,6 +16,8 @@ generic-service:
     LICENCE_API_URL: "https://create-and-vary-a-licence-api-dev.hmpps.service.justice.gov.uk"
     PRISONER_SEARCH_API_URL: "https://prisoner-offender-search-dev.prison.service.justice.gov.uk"
     PROBATION_SEARCH_API_URL: "https://probation-offender-search-dev.hmpps.service.justice.gov.uk"
+    EXIT_SURVEY_LINK: "#"
+    PHASE_BANNER_LINK: "#"
```

Comment:
I think we have an actual surveymonkey link available to us? Also, are these not going to be the same link?

Reply:
Not sure if they are the same link? Thought they were different i.e. one related to the create licence journey the other more generally about the beta service. I'll ask..

Reply:
Added the links given.